applications/nrf_desktop/configuration/nrf54lm20pdk_nrf54lm20a_cpuapp/prj_release.conf

```diff
@@ -40,6 +40,11 @@ CONFIG_DESKTOP_USB_ENABLE=y
 # Enable the only USB stack that supports USBHS
 CONFIG_DESKTOP_USB_STACK_NEXT=y
 
+# Suspend Bluetooth advertising when USB is connected to improve
+# USBHS report rate.
+CONFIG_DESKTOP_BLE_ADV_CTRL_ENABLE=y
```

Comment:
Make sure to also align `llvm` configuration after https://github.com/nrfconnect/sdk-nrf/pull/23000 goes in